### ISSUE TITLE
Harmony of Despair - LLVM Linkage Failed

### ISSUE THREAD

**Issue by JacobAL** [RPCS3/rpcs3#4217]
When playing with both recompilers, pressing start on the pause screen causes a white screen and emulation stops. 
Game works with PPU decoder set to interpreter (fast) though.

Log: https://www.dropbox.com/s/d8ee03mdpbai6o8/RPCS3.7z?dl=0
Specs: https://i.imgur.com/cwrtawj.png

The console says:
S {PPU[0x1000003] Thread () [0x00d159ac]} PPU: LLVM: Loaded module v2-dllregionfire.ppu.sprx+03FE88-B0E6D7C956A1C96F-haswell.obj
E PPU: LLVM: [0x103b0bc] Unsupported relocation(11) in 'dllregionfire.ppu.sprx'. Please report.
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0xb1ac
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x9d8
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x940
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x3af14
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0xd4ec
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0xa6d8
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0xa1c
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0xce10
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0xb2d4
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x350a4
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x247a0
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x3a560
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0xa68
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0xa74c
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x2482c
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x98c
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x2474c
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x3dc7c
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x1a0e0
E {PPU[0x1000003] Thread () [0x00d159ac]} LLVM: Linkage failed: __0x254f8
S {PPU[0x1000003] Thread () [0x00d159ac]} sys_prx: Loaded module: /dev_hdd0/game/NPUB30505/USRDIR/dlc/dlc02_stage_gekkaunder/dll/dllshot/dllregionfire.ppu.sprx

**Comment by scribam** [comment]
@JacobAL I think this issue has been fixed with PR #4278. Can you confirm?

**Comment by JacobAL** [comment]
Yes. Played it with recompilers a few weeks ago. Works fine but takes a while to start up initially.

**Comment by scribam** [comment]
Nice! Feel free to close this issue :smiley:
